TestCases/rans/flatplate/roughness/bc_wilcox2006/turb_SST_flatplate_roughBCWilcox2006.cfg

```diff
@@ -0,0 +1,248 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: Turbulent flow over flat plate with zero pressure gradient %
+% Author: Thomas D. Economon                                                   %
+% Institution: Stanford University                                             %
+% Date: 2011.11.10                                                             %
+% File Version 5.0.0 "Raven"                                                %
```

Comment:
You can probably grab and modify our most recent config for a flatplate to make it easier to cleanup the files